ably/proto_message_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"testing"
 
 	"github.com/ably/ably-go/ably"
+	"github.com/ably/ably-go/ablytest"
```

Comment:
If `ablytest` package is imported here, this file which previously only contained unit tests will now contain some integration tests. 

This file still has the build tags to run in CI at the unit test stage. We should probably leave this file alone as a place for unit tests and create a new file `proto_message_integration_test.go` for the new tests added in this PR so it can have the build tags for the integration stage of CI 
```
//go:build !unit
// +build !unit
```

Reply:
Done, thanks.